src/CommonCartridge.js

```diff
@@ -449,382 +469,398 @@ export default class CommonCartridge extends Component {
               </View>
             )}
 
-            <div style={{ marginTop: this.props.compact ? "0" : "12px" }}>
-              <Flex>
-                {this.state.showcaseSingleResource === null && (
-                  <FlexItem align="start" shrink>
-                    <nav>
-                      {this.state.modules.length > 0 && (
-                        <NavLink exact className="MenuItem" to="/">
-                          <Trans>Modules ({this.state.modules.length})</Trans>
-                        </NavLink>
-                      )}
-                      {numberOfAssignments > 0 && (
-                        <NavLink
-                          className="MenuItem"
-                          to={{
-                            pathname: "/assignments",
-                            search: startIndexQuery
-                          }}
-                        >
-                          <Trans>Assignments ({numberOfAssignments})</Trans>
-                        </NavLink>
-                      )}
-                      {this.state.pageResources.length > 0 && (
-                        <NavLink
-                          className="MenuItem"
-                          to={{
-                            pathname: "/pages",
-                            search: startIndexQuery
+            <div
+              className={[
+                "CommonCartridge--view",
+                singleResourceView ? "single-resource" : null
+              ]
+                .join(" ")
+                .trim()}
+            >
+              {this.state.showcaseSingleResource === null && (
+                <nav>
+                  <input type="checkbox" id="NavTrigger" name="NavTrigger" />
+                  <Button
+                    as="label"
+                    variant="ghost"
+                    buttonRef={ref =>
```

Comment:
This seems like it does not belong here. Why don't you add a className='open' and for='NavTrigger' props to the Button?

Reply:
Is just that className does not work with instui so the workaround is to use the ref instead